www/guides/react.md

```diff
@@ -0,0 +1,373 @@
+---
```

Comment:
(Don't know where to put this) when I was searching for the guide via [https://snowpack-git-react-guide.pikapkg.vercel.app/](https://snowpack-git-react-guide.pikapkg.vercel.app/), finding the guide was more difficult than I expected. Perhaps there could be a list of guides on the sidebar on the main page, which would link to the appropriate guide?